app/services/load_test.py

```
@@ -0,0 +1,411 @@
+            logger.error(f"Invalid data type: {type(data)}")
+            return LoadTestResult(
+                success=False,
+                status=LoadTestStatus.FAILED,
+                message="Invalid request data format",
+                error_details={"error_code": "INVALID_REQUEST_FORMAT"}
+            )
+
+        async with self._operation_context(operation_name, user_claims.sub, data.repo_alias_name) as operation_id:
+
+            logger.info(
+                f"Processing load test request",
```

Ruff (0.13.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)